OpenRA.Platforms.Default/Sdl2Input.cs

```diff
@@ -161,7 +161,11 @@ public void PumpInput(IInputHandler inputHandler)
 								Platform.CurrentPlatform == PlatformType.Windows)
 								Game.Exit();
 							else
-								inputHandler.OnKeyInput(keyEvent);
+							{
+								// Only register initial key press
+								if (e.key.repeat == 0)
```

Comment:
@pchote: how about replacing this with `public bool KeyInput.Repeated` and have `Hotkey.FromKeyInput(KeyInput ki)` return `Hotkey.Invalid` if `ki.Repeated` is true?